packages/java-parser/src/unicodesets.js

```diff
@@ -2236,7 +2245,7 @@ const mac_a = [
   [65056, 65071],
   [48, 57]
 ];
-f(ricd, mac_a);
+addRanges(ricd, mac_a);
 const ric = new Set(
```

Comment:
combining the sets takes **half** the execution file of this script (once the range adding function is optimized).
Is there a faster way to do so?

Reply:
Since we are only calling "has" with the sets, what I am thinking of is replacing ```ric```, that "fake" a set like so:
```
const ric = {
  has: (elt) => {
    if(fic.has(elt)) {
      return true;
    }
    return ricd.has(elt);
  }
};
```

Reply:
that may speed up the init time but at a cost of the actual runtime of parsing source code.